config/areas/site/buttons.php

```diff
@@ -25,7 +25,7 @@
 	// `languages` button needs to be in site area, as languages area itself
 	// is only loaded when in multilang setup
 	'languages' => function () {
-		return new LanguagesButton();
+		return new ContentTranslationsButton();
```

Comment:
`Reasoning` Could you explain the change here as well?

Reply:
Mainly cause I thought with the classes we could be a but more explicit, while I get that we use very short strings for referencing them in blueprints etc. 

I was talking to Lukas that merging the v4.3.1 changes into v5 would require adding proper languages button classes. And he thought the `LanguagesButton` class would already handle those given the name. So with that background and then adding `LanguagesAddButton` etc., the class name `LanguagesButton` felt a little out of place and non-descriptive.

Reply:
What about LanguagesDropdown or LanguagesToggle instead? I agree that we can/should be more explicit, but as long as we don't use ContentTranslations in more places, I would not personally go this way.

Reply:
Went for LanguagesDropdown

Reply:
Sorry, one more round 🙈 I hope I'm not too much a PITA at this point. Should it be LanguagesDropdownButton instead?